src/m365/spo/commands/serviceprincipal/serviceprincipal-permissionrequest-list.ts

```diff
@@ -58,6 +60,32 @@ class SpoServicePrincipalPermissionRequestListCommand extends SpoCommand {
       this.handleRejectedPromise(err);
     }
   }
+
+  private async getoAuth2Permissiongrants(sPOClientExtensibilityWebApplicationPrincipalId: string): Promise<string[]> {
+    const requestOptions: CliRequestOptions = {
+      url: `https://graph.microsoft.com/beta/oAuth2Permissiongrants/?$filter=clientId eq '${sPOClientExtensibilityWebApplicationPrincipalId}' and consentType eq 'AllPrincipals'`,
```

Comment:
`oAuth2Permissiongrants` is available on the v1.0 Graph endpoint too. Have you checked if we could use it from v1.0?

Reply:
Goog catch, that's indeed the case